build_run_specs_full.py

```diff
@@ -0,0 +1,237 @@
+    {'scenario':'social_studies','description': "mmlu:model=neurips/local,subject=high_school_geography,data_augmentation=canonical", 'priority': 2},
+    {'scenario':'social_studies','description': "mmlu:model=neurips/local,subject=us_foreign_policy,data_augmentation=canonical", 'priority': 2},
+    {'scenario':'social_studies','description': "mmlu:model=neurips/local,subject=public_relations,data_augmentation=canonical", 'priority': 2},
+    {'scenario':'social_studies','description': "mmlu:model=neurips/local,subject=security_studies,data_augmentation=canonical", 'priority': 2},
+
+    # MMLU Social Sciences: Human behavior
+    {'scenario':'human_behavior','description': "mmlu:model=neurips/local,subject=high_school_psychology,data_augmentation=canonical", 'priority': 2},
+    {'scenario':'human_behavior','description': "mmlu:model=neurips/local,subject=human_sexuality,data_augmentation=canonical", 'priority': 2},
+    {'scenario':'human_behavior','description': "mmlu:model=neurips/local,subject=professional_psychology,data_augmentation=canonical", 'priority': 2},
+    {'scenario':'human_behavior','description': "mmlu:model=neurips/local,subject=sociology,data_augmentation=canonical", 'priority': 2},
+
+    # MMLU Social Sciences: Economics
+    {'scenario':'economics','description': "mmlu:model=neurips/local,subject=high_school_microeconomics,data_augmentation=canonical", 'priority': 2},
+    {'scenario':'economics','description': "mmlu:model=neurips/local,subject=econometrics,data_augmentation=canonical", 'priority': 2},
+    {'scenario':'economics','description': "mmlu:model=neurips/local,subject=high_school_macroeconomics,data_augmentation=canonical", 'priority': 2},
+    
+    # Truthful QA
+    {'scenario':'truthful_qa','description': "truthful_qa:task=mc_single,model=neurips/local", 'priority': 1},
+
+    # CNN/daily mail
+    {'scenario':'truthful_qa','description': "summarization_cnndm:model=neurips/local", 'priority': 1},
+    # GSM
+    {'scenario':'gsm','description': "gsm:model=neurips/local", 'priority': 1},
+    # BBQ
+    {'scenario':'bbq','description': "bbq:subject=all,model=neurips/local", 'priority': 1},
+
+]
+
+def generate_equal_sum_list(V, N):
+    # Calculate the base value that will be repeated.
+    base_value = V // N
+    # Calculate the remainder for distribution.
+    remainder = V % N
+    
+    # Create the list with base_value repeated N times.
+    result = [base_value] * N
+    
+    # Distribute the remainder evenly among the elements.
+    for i in range(remainder):
+        result[i] += 1
+    
+    return result
+
+
+if __name__ == "__main__":
+
+    import pandas as pd
+    df = pd.DataFrame(entries)
+    scenarios = df.value_counts('scenario').to_dict()
+
+    total_n_examples = 500
+    max_eval_instances_per_scenario = total_n_examples//len(df.scenario.unique())
```

Comment:
You will have fewer than `total_n_examples` if the division has a remainder, which is the case for both 500 and 600. 
For example, when running with `total_n_examples` of 600, you will end up with 570 total examples. Maybe it's worth distributing remainder somehow like you were doing earlier, or change the name of the file to the actual number of examples that will be run.

Reply:
good idea, I will distribute these